packages/core/nuxt-theme-module/theme/layouts/error.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div id="error">
+    <SfImage
+      class="image"
+      :src="require('@storefront-ui/shared/icons/error.svg')"
+    />
+    <SfHeading
+      title="Page not found"
+      :level="2"
+      description="We are sorry that we can’t find the page, please go back or try again"
+      class="heading sf-heading--no-underline"
+    />
+    <div class="actions">
+      <SfButton link="/" class="sf-button--full-width actions__button">
+        Return home
+      </SfButton>
+      <SfButton class="sf-button--full-width sf-button--text actions__button">
+        Back
```

Comment:
We should go back here
https://router.vuejs.org/guide/essentials/navigation.html#router-go-n